spec/unit/installer/xcode/target_validator_spec.rb

```diff
@@ -322,69 +321,82 @@ def create_validator(sandbox, podfile, lockfile)
             'the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.'
           end
 
-          it 'does not raise when targets integrate the same pod but only one of the targets is a swift target' do
+          it 'does not raise when integrating targets is disabled' do
```

Comment:
diff is weird but this is the new test here that explicitly states `integrate_targets => false`